package.json

```diff
@@ -26,6 +26,8 @@
   ],
   "dependencies": {
     "@babel/runtime": "^7.22.6",
+    "alpinejs": "^3.13.2",
```

Comment:
`alphinejs` and `codemirror` should be part listed in the package.json of the example, not of mathjs itself.

Reply:
It should be resolved now.